Comments/Models/Context.cs

```diff
@@ -270,7 +270,7 @@ public void UpdateCommentStatus(IEnumerable<int> commentIds, Status status)
 	    {
 		    var commentsToUpdate = Comment.Where(c => commentIds.Contains(c.CommentId)).ToList();
 
-		    if (commentsToUpdate.Any(c => c.CreatedByUserId != _createdByUserID))
+		    if (commentsToUpdate.Any(c => (c.CreatedByUserId != _createdByUserID && c.OrganisationId != _organisationalLeadOrganisationID)))
```

Comment:
i don't think this is quite right. does this support the changes in the confluence doc?